src/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ pkg_check_modules(GOBJECT REQUIRED gobject-2.0)
 pkg_check_modules(GIO REQUIRED gio-2.0)
 pkg_check_modules(GEE REQUIRED gee-0.8)
 pkg_check_modules(POPPLER REQUIRED poppler-glib)
-pkg_check_modules(GTK REQUIRED gtk+-3.0>=3.10)
+pkg_check_modules(GTK REQUIRED gtk+-3.0>=3.18)
```

Comment:
FYI, Debian stable has 3.14.5. Stretch will probably appear sometime in the next months (and has 3.22) but until then, Debian users won't be able to use the latest pdfpc anymore.